docs/contributing/coding-standards/css.md

```diff
@@ -167,144 +167,124 @@ Good:
 
 Bad:
 
-```
+```scss
 $white: #FFF;
 ```
 
 Good:
 
-```
+```scss
 $white: #ffffff;
 ```
 
 ### Avoid using ID selectors
 
 Bad:
 
-```
+```scss
 #content {
-  ...
+  // ...
 }
 ```
 
 Good:
 
-```
+```scss
 .govuk-wrapper {
-  ...
-}
-```
-
-### Use single colons for pseudo-element selectors
```

Comment:
Removed this guidance as it's no longer necessary since https://github.com/alphagov/govuk-frontend/commit/f9c7ff0cfa5466076870275827d96e0554e4337d